test/integration/node-specific/client_close.test.ts

```diff
@@ -584,7 +699,57 @@ describe.skip('MongoClient.close() Integration', () => {
 
   describe('Server resource: Cursor', () => {
     describe('after cursors are created', () => {
-      it.skip('all active server-side cursors are closed by client.close()', async function () {});
+      let client;
+      let coll;
+      let cursor;
+      let utilClient;
+
+      beforeEach(async function () {
+        client = this.configuration.newClient();
+        utilClient = this.configuration.newClient();
+        await client.connect();
+        coll = await client.db('db').collection('coll', { capped: true, size: 1_000_000 });
```

Comment:
Did you mean to use "createCollection" here? this doesn't actually create a capped collection

Make sure you drop it before creating it (and you squash the ns not found error from drop)

Reply:
Sounds good!